src/oscar/apps/catalogue/abstract_models.py

```diff
@@ -613,6 +623,20 @@ def get_title(self):
         return title
     get_title.short_description = pgettext_lazy("Product title", "Title")
 
+    def get_meta_title(self):
+        title = self.meta_title
+        if not title and self.is_child:
+            title = self.parent.meta_title
```

Comment:
Why not add a fallback to the product title here?

Reply:
@specialunderwear this is the `Category` model, not product.

Reply:
@specialunderwear we do fall back to the title, below, but I think parent meta title should take precedence over child title?

Reply:
ok

Reply:
but also fall back to the parent title if the title is empty as well?

Reply:
Yes - `get_title()` will handle the fallback to parent title.

Reply:
ok